ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -159,14 +160,19 @@ private OptimizingQueue getQueueByToken(String token) {
 
   @Override
   public List<OptimizerInstance> listOptimizers() {
-    return optimizingQueueByGroup.values().stream()
+    return optimizingQueueByGroup.values()
+        .stream()
         .flatMap(queue -> queue.getOptimizers().stream())
+        .sorted(Comparator.comparingLong(OptimizerInstance::getStartTime).reversed())
         .collect(Collectors.toList());
   }
 
   @Override
   public List<OptimizerInstance> listOptimizers(String group) {
-    return getQueueByGroup(group).getOptimizers();
+    return getQueueByGroup(group).getOptimizers()
```

Comment:
I don't think the service layer needs to guarantee the order of results. The order of results should be flexibly set by the Controller layer itself.

Reply:
ok, i will resolve